### PR TITLE
Fix developer guide bugs

### LINKED ISSUE #227: DG bugs
1. Model is outdated
image

2. Font size for images are too small

3. Outdated areas
image
* we have no sample friends loaded

4. Instructions for manual testing
image
* Is listing all friends really still a prerequisite? Since our app is no longer indexed based, I would argue that simply entering a valid FRIEND_ID is the prerequisite.

5. Number beside title of Viewing a game's full data and a slash on point 5 (same problem in the above section due to copy paste)
image

### PR DESCRIPTION
**Pull request by MarcusTXK**
Changes:
* Fix some bugs found in the developer guide

Issues:
* Resolves #227

### CODE REVIEW

**Review by tau-bar — APPROVED** [review]
LGTM, minor nits.

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -476,10 +476,9 @@ testers are expected to do more *exploratory* testing.
...
 1. Initial launch
 
-   1. Download the jar file and copy into an empty folder
+   1. Download the jar file and copy into an empty folder.
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample friends. The window size may not be 
-      optimum.
+   1. Double-click the jar file Expected: gitGud GUI launches.
 
 1. Saving window preferences
```
Comment (tau-bar) [inline docs/DeveloperGuide.md:484]: Can you help remove `the resize window to an optimum size`. We can't resize it anymore.

**Review by tau-bar — CHANGES_REQUESTED** [review]
LGTM. minor nits to fix

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -505,8 +501,6 @@ testers are expected to do more *exploratory* testing.
     4. Other incorrect add commands to try: `friend --name Marcus`, `friend --add`, `friend --name`, ...`
```
Comment (tau-bar) [inline docs/DeveloperGuide.md:501]:
```suggestion
    4. Other incorrect add commands to try: `friend --name Marcus`, `friend --add`, `friend --name`, `...`
```

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -540,8 +534,6 @@ testers are expected to do more *exploratory* testing.
        Draconian`, `link --friend`, ...
```
Comment (tau-bar) [inline docs/DeveloperGuide.md:534]: There is inconsistency with these `...`, some are differently formatted. Perhaps they are not even necessary. Can you help to remove or make consistent?

Comment (MarcusTXK) [reply docs/DeveloperGuide.md]: I agree, I shall remove them all.

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -505,8 +494,6 @@ testers are expected to do more *exploratory* testing.
     4. Other incorrect add commands to try: `friend --name Marcus`, `friend --add`, `friend --name`, ...`
```
Comment (tau-bar) [inline docs/DeveloperGuide.md:494]:
```suggestion
    4. Other incorrect add commands to try: `friend --name Marcus`, `friend --add`, `friend --name`
```

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -645,13 +624,5 @@ testers are expected to do more *exploratory* testing.
     4. Test case: `game --get rant`<br>
        Expected: No game is found. Error details shown in the status message. Status bar remains the same.
 
-    5. \Other incorrect delete commands to try: `game --get`, `...`
+    5. Other incorrect delete commands to try: `game --get`
```
Comment (tau-bar) [inline docs/DeveloperGuide.md:627]:
```suggestion
    5. Other incorrect get commands to try: `game --get`
```

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -627,10 +606,10 @@ testers are expected to do more *exploratory* testing.
     4. Test case: `friend --get co`<br>
        Expected: No friend is found. Error details shown in the status message. Status bar remains the same.
 
-    5. \Other incorrect delete commands to try: `friend --get`, `...`
+    5. Other incorrect delete commands to try: `friend --get`
```
Comment (tau-bar) [inline docs/DeveloperGuide.md:609]:
```suggestion
    5. Other incorrect get commands to try: `friend --get`
```

**Review by tau-bar — APPROVED** [review]
LGTM.

**Pull request merged**